### DISCUSSION TITLE
Using the Touch pins ESP32-S2?

### DISCUSSION THREAD

**Discussion by RayEbb** [tttapa/Control-Surface#1056] [Q&A]
Hello! 

My name is Raymond (52), and I live in the Netherlands. Thank you for this beautiful and very interesting library! 👍 

Unfortunately I'm not a programmer, and partly because of this it's hard for me to find my way how to use this library, sorry. 

I bought an Esp32-S2 Wrover module from Olimex last week, as it supports USB Host/Device. The buttons, potentiometers and rotary encoder examples are functioning without any problems. 

Now for my (maybe silly) question. Is it possible to use the Touch pins of the ESP32-S2 in combination with, for example, a Control Change or MMC (Play, Stop...)?

Thank you very much in advance! 

Raymond

**Comment by tttapa** [comment]
Yes, but not out of the box. You can use the [Custom-MIDI-Output-Element.ino](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html) example as a starting point.

**Reply by RayEbb** [reply]
Thank you very much for the quick reply! 👍 I'll try this. Hopefully I won't have to bother you anymore. 🙏😉
